lib/src/mock.dart

```diff
@@ -1267,3 +1267,27 @@ extension ListOfVerificationResult on List<VerificationResult> {
   /// [verifyInOrder].
   List<List<dynamic>> get captured => [...map((result) => result.captured)];
 }
+
+final _defaultDummyValueFor = <Type, dynamic>{
```

Comment:
We should use `Object?` instead of `dynamic`. Nothing here requires dynamic dispatch.